### PR TITLE
Article CloudBoost Cache added. 

### PR DESCRIPTION
**Pull request by nawazdhandala**

### CODE REVIEW

File: articles/cloudboost-cache.markdown [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+### npm ###
+
+[npm](http://github.com/isaacs/npm) is the most popular package manager for Node.js.
+Installing npm is easy.
+
+    curl http://npmjs.org/install.sh | sh
+
+A note for windows users: npm does not currently work on windows. It will in the future.
+
```
Comment (creationix) [inline articles/cloudboost-cache.markdown]: npm has come with node core for quite some time and works fine on windows using the official msi installer.

Comment (nawazdhandala) [inline articles/cloudboost-cache.markdown]: Thank you so much @creationix for pointing it out. I almost always use my Ubuntu box for dev and did not realize this. :)

Comment (nawazdhandala) [inline articles/cloudboost-cache.markdown]: I have updated the article. Let me know if you need any more edits.

Comment (creationix) [inline articles/cloudboost-cache.markdown]: I would just remove the section and assume the user has npm installed.  I imagine most people get npm as part of their install install and if they don't, they already know how to install npm.

Comment (nawazdhandala) [inline articles/cloudboost-cache.markdown]: @creationix Good idea. Done!

**Comment by creationix** [comment]
Nice work.

**Comment by nawazdhandala** [comment]
@creationix  Thank you so much for the merge.

**Pull request merged**